emojidataios/Classes/EmojiCategory.swift

```diff
@@ -8,14 +8,15 @@
 
 import Foundation
 
-public enum EmojiCategory: String {
-  
-  case SYMBOLS = "Symbols"
-  case OBJECTS = "Objects"
-  case NATURE = "Animals & Nature"
-  case PEOPLE = "Smileys & People"
-  case FOODS = "Food & Drink"
-  case PLACES = "Travel & Places"
-  case ACTIVITY = "Activities"
-  case FLAGS = "Flags"
+public enum EmojiCategory: String, CaseIterable {
+	case SYMBOLS = "Symbols"
+	case OBJECTS = "Objects"
+	case NATURE = "Animals & Nature"
+	case PEOPLE = "Smileys & Emotion"
+	case BODY = "People & Body"
+	case FOODS = "Food & Drink"
+	case PLACES = "Travel & Places"
+	case ACTIVITY = "Activities"
+	case FLAGS = "Flags"
+	case FAV = "⭐️ Favorites" // intended to be used for a local db
```

Comment:
The library doesn't know what a DB is, or what it will be used for so you'll have to implement "Favorite" somewhere else